ipinfo/mmdb.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipinfo
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/oschwald/geoip2-golang"
+)
+
+// mmIPInfoMap is a [ipinfo.IPInfoMap] that uses [geoip2.Reader] to read MMDB files.
```

Comment:
```suggestion
// mmIPInfoMap is an [ipinfo.IPInfoMap] that uses [geoip2.Reader] to read MMDB files.
```

Reply:
Shall we also expand "MMDB" to "MaxMind DB" to be more searchable and accurate in Google?